packages/block-library/src/query-pagination-next/index.php

```diff
@@ -72,9 +72,9 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 			)
 		) ) {
 			$p->set_attribute( 'data-wp-key', 'query-pagination-next' );
-			$p->set_attribute( 'data-wp-on--click', 'actions.core.query.navigate' );
-			$p->set_attribute( 'data-wp-on--mouseenter', 'actions.core.query.prefetch' );
-			$p->set_attribute( 'data-wp-effect', 'effects.core.query.prefetch' );
+			$p->set_attribute( 'data-wp-on--click', 'core/query::actions.navigate' );
```

Comment:
I think for now it's fine to use the namespace manually here, althoug we need to think more about this. Maybe these blocks should also have a `data-wp-interactive` directive as well.